src/css/components/task-file-list.less

```diff
@@ -0,0 +1,22 @@
+/* ==============
+ * Task File List (table)
+ * ==============
+ */
+.task-file-list {
+  margin-top: @base-spacing-unit * 3;
+
+  tr {
+    .btn {
```

Comment:
Would it make sense to create a `.btn.reveal` class to handle the "show on hover" case? I find the nesting quite inexpressive in this case.